src/Common/dataAccess/createCollection.ts

```diff
@@ -182,6 +188,11 @@ const createMongoCollection = async (params: DataModels.CreateCollectionParams):
     params.collectionId,
     rpPayload
   );
+
+  TelemetryProcessor.trace(Action.CreateMongoCollectionWithWildcardIndex, ActionModifiers.Mark, {
```

Comment:
>trace [](start = 21, length = 5)

do we have telemetry for all mongo create operations here. so we can easily compute the ratio of people opting in with just the data we have?

Reply:
this was supposed to be only if we params had "createMongoWildcardIndex". will update it. this telemetry is only to trace mongo collection creation where user has opted in to have wild card index on all fields